scripts/openroad/pdn.tcl

```diff
@@ -32,7 +32,12 @@ if {[catch {read_def $::env(CURRENT_DEF)} errmsg]} {
     exit 1
 }
 
-if {[catch {pdngen $::env(PDN_CFG) -verbose} errmsg]} {
+if {[catch {convert_pdn_config $::env(PDN_CFG)} errmsg]} {
+    puts stderr $errmsg
+    exit 1
+}
+
+if {[catch {pdngen -verbose} errmsg]} {
```

Comment:
`-verbose` has been removed so you will just keep getting warnings about that if it's kept.

Reply:
Removed.